react/ProfileField.js

```diff
@@ -5,65 +5,66 @@ import RuleFieldShape from './RuleFieldShape'
 import ProfileFieldShape from './ProfileFieldShape'
 
 class ProfileField extends Component {
-  componentDidUpdate() {
-    const { field, data, onFieldUpdate } = this.props
-    if (data.focus && this.el) {
-      this.el.focus()
-      onFieldUpdate({ [field.name]: { ...data, focus: false } })
+    componentDidUpdate() {
+        const { field, data, onFieldUpdate } = this.props
+        if (data.focus && this.el) {
+            this.el.focus()
+            onFieldUpdate({
+                [field.name]: {...data, focus: false } })
+        }
     }
-  }
 
-  handleChange = e => {
-    const { field, data, onFieldUpdate } = this.props
-    const { value } = e.target
+    handleChange = e => {
+        const { field, data, onFieldUpdate } = this.props
+        const { value } = e.target
 
-    const error = data.touched ? applyValidation(field, value) : null
-    const maskedValue = applyMask(field, value)
+        const error = data.touched ? applyValidation(field, value) : null
+        const maskedValue = applyMask(field, value)
 
-    onFieldUpdate({ [field.name]: { ...data, value: maskedValue, error } })
-  }
-
-  handleBlur = () => {
-    const { field, data, onFieldUpdate } = this.props
-    const error = applyValidation(field, data.value)
+        onFieldUpdate({
```

Comment:
I think that there is a problem with the .editorconfig. It has changed from spaces to tabs, it makes harder to understand what was the change...

Reply:
Has the spacing been changed?

Reply:
@salesfelipe Correct the spacing leaving it as it was in the master.